cmd/internal/server/handlers/fivetran_value_converters.go

```diff
@@ -134,7 +134,8 @@ var converters = map[fivetransdk.DataType]ConverterFunc{
 		// The DATETIME type is used for values that contain both date and time parts.
 		// MySQL retrieves and displays DATETIME values in 'YYYY-MM-DD hh:mm:ss' format.
 		// The supported range is '1000-01-01 00:00:00' to '9999-12-31 23:59:59'.
-		t, err := time.Parse("2006-01-02 15:04:05", value.ToString())
+
+		t, err := time.Parse(time.DateTime, value.ToString())
```

Comment:
@Phanatic You missed a case just below this under `fivetransdk.DataType_UTC_DATETIME` which can also use this constant.